migrations.config.ts

```diff
@@ -23,6 +23,14 @@ const DRIVER_MAP: Record<DbEngine, () => Promise<Options['driver']>> = {
 
 const isProduction = process.env.NODE_ENV === 'production';
 
+function resolveDriverOptions(): Dictionary {
+  if ((process.env.SHLINK_DASHBOARD_DB_USE_ENCRYPTION ?? '') === 'true') {
```

Comment:
No need to coalesce to empty string here. If the value is `undefined` the condition will still evaluate to `false`.


```suggestion
  if (process.env.SHLINK_DASHBOARD_DB_USE_ENCRYPTION === 'true') {
```